snapshot/lib/snapshot/test_command_generator.rb

```diff
@@ -55,9 +55,17 @@ def verify_devices_share_os(devices)
         end
         # Return true if all devices are iOS devices
         return true unless all_ios.include?(false)
+
+        all_tvos = devices.map do |device|
+          device = device.downcase
+          device.start_with?('apple tv')
```

Comment:
Doesn't the `.map` method use the return value, so in this case the `all_tvos` array would be full of `[true, false, true, true]`?

Reply:
Indeed, `all_tvos` is an array of boolean (`true` if a device starts with `apple tv`).  After that, the method will return `true` if `all_tvos` is full of `true` (i.e. all devices are `apple tv`).